datalad/tests/test_annexrepo.py

```diff
@@ -48,6 +52,27 @@ def test_AnnexRepo_instance_from_existing(path):
 @with_tempfile
 def test_AnnexRepo_instance_brand_new(path):
 
-    gr = AnnexRepo(path)
-    assert_is_instance(gr, AnnexRepo, "AnnexRepo was not created.")
-    assert_true(os.path.exists(os.path.join(path, '.git')))
\ No newline at end of file
+    ar = AnnexRepo(path)
+    assert_is_instance(ar, AnnexRepo, "AnnexRepo was not created.")
+    assert_true(os.path.exists(os.path.join(path, '.git')))
+
+
+
+@with_testrepos
```

Comment:
yeah -- also we need to either create funky test repos with some obscured filenames, or generate them using with_tree as I did in other tests